src/scripts/browserDeviceProfile.js

```diff
@@ -346,26 +380,19 @@ import browser from './browser';
         const canPlayAc3VideoAudioInHls = supportsAc3InHls(videoTestElement);
 
         // Transcoding codec is the first in hlsVideoAudioCodecs.
-        // Prefer AAC, MP3 to other codecs when audio transcoding.
-        if (canPlayAacVideoAudio) {
-            videoAudioCodecs.push('aac');
-            hlsInTsVideoAudioCodecs.push('aac');
-            hlsInFmp4VideoAudioCodecs.push('aac');
-        }
-
+        // AAC, MP3, AC-3 and EAC-3 are supported in HLS
+        // Here they are sorted based on the quality of their encoders in ffmpeg
         if (supportsMp3VideoAudio) {
             videoAudioCodecs.push('mp3');
 
-            // PS4 fails to load HLS with mp3 audio
-            if (!browser.ps4) {
+            // PS4 fails to load HLS with MP3 audio
+            // Only prefer MP3 when the client wants stereo or mono to prevent downmixing
+            if (!browser.ps4 && physicalAudioChannels <= 2) {
                 hlsInTsVideoAudioCodecs.push('mp3');
+                hlsInFmp4VideoAudioCodecs.push('mp3');
```

Comment:
Do we know if the PS4 supports fMP4 and if MP3 works in fMP4 on a PS4 or has the same issue as HLS?

Reply:
Don't have a PS4, the documentation (https://www.sie.com/content/dam/corporate/jp/guideline/PS4_Web_Content-Guidelines_e.pdf) states it only supports this specification: https://tools.ietf.org/html/draft-pantos-http-live-streaming-05 which didn't specify fmp4 yet. If it does it wouldn't be the first time their documentation was lacking, though.